### PR TITLE
userstory #2

### PR DESCRIPTION
**Pull request by Malte1983**
- add function to create a new Task and store them in localstorage 
## Resolve 

- userstory #2 
- add delete function
- add checked tasks function

### CODE REVIEW

**Comment by Malte1983** [comment]
Sorry, i accidentally merged first pull request from userstory 02 😞

File: src/App.js [hunk 1]
```diff
@@ -44,19 +44,20 @@ function App() {
...
   )
   function handleCreateTasks({ todo }) {
     const newTasks = [
-      ...tasks,
       {
         id: uuidv4(),
         todo: todo,
         completed: false,
       },
+      ...tasks,
```
Comment (macfrei) [inline src/App.js:60]: WHoop! 🥳

File: src/App.js [hunk 2]
```diff
@@ -73,6 +74,13 @@ function App() {
     setTasks(newTasks)
     localStorage.setItem('tasksLocalStorage', JSON.stringify(newTasks))
   }
+
+  function handleDeleteTask(id) {
+    const filteredData = tasks.filter(task => task.id !== id)
+    const stringifiedValue = JSON.stringify(filteredData)
+    localStorage.setItem('tasksLocalStorage', stringifiedValue)
+    setTasks(filteredData)
+  }
```
Comment (macfrei) [inline src/App.js:83]: Nice 👍

**Review by macfrei — COMMENTED** [review]
Nice job Malte! Do you have a `ToDoCard.spec.js` where you test your component? And what about storybook? 😉

**Pull request merged**